scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -43,7 +43,9 @@ class FormatWriter(formatOps: FormatOps) {
                 .getOrElse(token.syntax, token.syntax)
             sb.append(rewrittenToken)
         }
-        sb.append(whitespace)
+
+        handleTrailingCommasAndWhitespace(formatToken, state, sb, whitespace)
```

Comment:
I had to incorporate the insertion of the whitespace, since in one case it needs to be skipped, namely when rewriting

```scala
foo(a,
  b,
)
```

to

```scala
foo(a, b)
```

Inserting the white space would result in

```scala
foo(a, b )
```

instead